src/logger.ts

```diff
@@ -101,6 +101,7 @@ class Logger {
     return this;
   }
 
+  /** Add custom CSS styles */
```

Comment:
I would add a short description of what the argument is
```js
/**
  * Add custom CSS styles
  * @param a string of CSS, similar to what you'd use for <div style="">
  */
```

Reply:
/**
  * Add custom CSS styles
  * @param
  * a string of CSS, similar to what you'd use for <div style="">
  */

Will this work better? Otherwise @param will take a as a variable

Reply:
good catch ![](https://hips.hearstapps.com/ghk.h-cdn.co/assets/cm/15/11/54ffe5266025c-dog1.jpg)